packages/starknet-snap/src/estimateFee.ts

```diff
@@ -37,6 +37,8 @@ export async function estimateFee(params: ApiParamsWithKeyDeriver) {
     );
     const { senderAddress } = requestParamsObj;
     const network = getNetworkFromChainId(state, requestParamsObj.chainId);
+    const transactionVersion =
+      requestParamsObj.transactionVersion ?? TRANSACTION_VERSION;
```

Comment:
is it more simple to put this in the estimateFeeBulk instead of put it in individual RPC?